### DISCUSSION TITLE
请问采用PaddleSpeech怎么实现实时语音识别？

### DISCUSSION THREAD

**Discussion by dabaoyasu** [PaddlePaddle/PaddleSpeech#1188] [Q&A] [answered]
比如，针对电话的应用场景，对通话过程中做同声翻译，需要进行语音流的实时识别，应该怎么调用？

**Comment by zh794390558** [comment]
先有电话的ASR。已DS2模型为例，流式部分参考https://github.com/PaddlePaddle/PaddleSpeech/blob/develop/paddlespeech/s2t/exps/deepspeech2/model.py#L402

**Reply by dabaoyasu** [reply]
语音流的断句需要在应用层自己判断吗？由应用层断句后生成一句话的语音文件，再调用deepspeech2的接口进行识别？

**Reply by FredGoo** [reply]
感谢

**Comment by zh794390558** [comment, marked as answer]
目前已经在 server 目录下实现了ws的流式asr，效果还在调试中。可以先体验下。
